phing/tasks/setup.xml

```diff
@@ -248,13 +248,17 @@
           <param>drush</param>
         </drush>
         <!-- Revert all features. -->
-        <drush command="fra" assume="yes" alias="${drush.alias}">
-          <option name="bundle">${cm.features.bundle}</option>
-        </drush>
+        <foreach list="${cm.features.bundle}" param="bundle" target="setup:update:features-import" />
```

Comment:
@timcosgrove I believe that this changes requires cm.features.bundle to be an array (it can never b e a string).

If that is the case, we should change the variable name to cm.features.bundles and change the example here to be an array.